tfhe-benchmark/src/params.rs

```diff
@@ -393,6 +398,155 @@ pub mod shortint_params {
+            self.compression
+        }
+
+        pub fn cast(&self) -> Option<ShortintKeySwitchingParameters> {
+            self.cast
+        }
+
+        pub fn noise_squash(&self) -> Option<NoiseSquashingParameters> {
+            self.noise_squash
+        }
+
+        pub fn noise_squash_compression(&self) -> Option<NoiseSquashingCompressionParameters> {
+            self.noise_squash_compression
+        }
+    }
+
+    struct ParamsGroupBuilder {
+        params_group: crate::params::ParamsGroup,
+    }
+
+    impl ParamsGroupBuilder {
+        pub fn new(base_params: PBSParameters) -> Self {
+            Self {
+                params_group: crate::params::ParamsGroup {
+                    base: base_params,
+                    cpke: None,
+                    compression: None,
+                    cast: None,
+                    noise_squash: None,
+                    noise_squash_compression: None,
+                },
+            }
+        }
+
+        pub fn cpke(mut self, params: CompactPublicKeyEncryptionParameters) -> Self {
+            self.params_group.cpke = Some(params);
+            self
+        }
+
+        pub fn compression(mut self, params: CompressionParameters) -> Self {
+            self.params_group.compression = Some(params);
+            self
+        }
+
+        pub fn cast(mut self, params: ShortintKeySwitchingParameters) -> Self {
+            self.params_group.cast = Some(params);
+            self
+        }
+
+        pub fn noise_squash(mut self, params: NoiseSquashingParameters) -> Self {
+            self.params_group.noise_squash = Some(params);
+            self
+        }
+
+        pub fn noise_squash_compression(
+            mut self,
+            params: NoiseSquashingCompressionParameters,
+        ) -> Self {
+            self.params_group.noise_squash_compression = Some(params);
+            self
+        }
+
+        pub fn build(self) -> crate::params::ParamsGroup {
+            self.params_group
+        }
+    }
+
+    pub fn get_classical_tuniform_groups() -> Vec<crate::params::ParamsGroup> {
+        // TODO Complete parameters groups once corresponding parameters set are available.
+        vec![
+            ParamsGroupBuilder::new(BENCH_PARAM_MESSAGE_1_CARRY_1_KS_PBS_TUNIFORM_2M128.into())
+                .build(),
+            ParamsGroupBuilder::new(BENCH_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128.into())
+                .cpke(BENCH_PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
+                .compression(BENCH_COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
+                .cast(BENCH_PARAM_KEYSWITCH_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
+                .noise_squash(BENCH_NOISE_SQUASHING_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
+                .noise_squash_compression(
+                    BENCH_NOISE_SQUASHING_COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128,
+                )
+                .build(),
```

Comment:
For GPU we should have something equivalent:

```
ParamsGroupBuilder::new(BENCH_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128.into())
                .cpke(BENCH_PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
                .compression(BENCH_COMP_PARAM_GPU_MULTI_BIT_GROUP_4_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
                .cast(PARAM_GPU_MULTI_BIT_GROUP_4_KEYSWITCH_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
                .noise_squash(BENCH_NOISE_SQUASHING_PARAM_GPU_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128)
                .noise_squash_compression(
                    BENCH_NOISE_SQUASHING_COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M128,
                )
                .build(),
```

Reply:
So no changes at all for now ?